src/update-status.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line
+import { getAllTodos } from './todo';
+
+export const updateToFalse = (todo) => {
+  const localStoragetodos = getAllTodos();
+  const index = localStoragetodos.findIndex((td) => td.index === todo.index);
+  localStoragetodos[index].completed = false;
+  localStorage.setItem('todos', JSON.stringify(localStoragetodos));
+};
+
+export const updateToTrue = (todo) => {
+  const localStoragetodos = getAllTodos();
+  const index = localStoragetodos.findIndex((td) => td.index === todo.index);
+  localStoragetodos[index].completed = true;
+  localStorage.setItem('todos', JSON.stringify(localStoragetodos));
+};
+
+const btnClear = document.getElementById('deleteAllCompleted');
+// eslint-disable-next-line
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.